packaging/crio.conf.d/microshift_arm64.conf

```diff
@@ -0,0 +1,18 @@
+[crio.runtime]
+selinux = true
+conmon = ""
+conmon_cgroup = "pod"
+cgroup_manager = "systemd"
+
+[crio.network]
+# rhel8 crio is configured to only look at /usr/libexec/cni, we override that here
+plugin_dirs = [
+        "/usr/libexec/cni",
+        "/opt/cni/bin"
+]
+
+# the pull secret is mandatory for MicroShift builds on top of OpenShift
+# for community builds on top of OKD, this setting has no effect
+[crio.image]
+global_auth_file="/etc/crio/openshift-pull-secret"
+pause_image = "registry.k8s.io/pause:3.6"
```

Comment:
Shouldn't we add the OCP image here already?

Reply:
fixed.